bus/bus.go

```diff
@@ -30,6 +31,21 @@ import (
 	"go.uber.org/zap"
 )
 
+// Client re-exports the client from the client package.
+type Client struct {
+	*client.Client
+}
+
+// NewClient returns a new bus client.
+func NewClient(addr, password string) *Client {
+	return &Client{
+		client.NewClient(
```

Comment:
(nit) would call this `client.New` for the sub package.

Reply:
that is not a `nit` sir